### DISCUSSION TITLE
Support for Ubuntu 24.04 LTS?

### DISCUSSION THREAD

**Discussion by StormTrooper** [inventree/InvenTree#7168] [Q&A] [answered]
With the release of Ubuntu 24.04 LTS last month is there any particular reason why the the installer still refers to only Ubuntu 20.04 as supported?

20.04 has about a year of support left, so not that long. 

Are these any dependency issues with later Ubuntu versions or do I need to stick to 20.04 for the meantime?
(I might fire up a test server and see how far it gets with a updated installer script)

**Comment by matmair** [comment]
Feel free to test, validate and support 24.04 LTS for the support life of 24.04

**Comment by SergeoLacruz** [comment]
I use 22.04  sinc it came out without any issues

**Reply by StormTrooper** [reply]
Did  you do a manual install? I apt packages are only available to 20.04 so I'm guessing so.

**Reply by SergeoLacruz** [reply]
yes, bare metal.

**Comment by rohitsingal1** [comment]
Anything on this. I have recently upgraded to Ubuntu 22.04. Trying bare metal installation and I get the error "This OS is currently not supported.".

**Reply by matmair** [reply]
I think my last post on this topic is still valid

**Reply by rohitsingal1** [reply]
How do I test it if I cannot install it ?

**Reply by SergeoLacruz** [reply]
The docs tell everything you need
https://docs.inventree.org/en/stable/start/install/
You can run it manually without the installer.

**Reply by StormTrooper** [reply, marked as answer]
You can manually install it but then can't use apt to easily keep it up to date. Updates will have to be manually installed. 

A while back I asked packager.io to add support for Ubuntu 24.04 which they added so will take another stab and see if I can get it working.
